src/sdl2/video.rs

```diff
@@ -571,6 +571,18 @@ impl VideoSubsystem {
         }
     }
 
+    pub fn display_dpi(&self, display_index: i32) -> Result<(f32, f32, f32), String> {
```

Comment:
I might be a little picky but wouldn't it be nicer if you could add some doc about what the 3 float values are exactly ? 

From `cargo doc` we can only see that it outputs a Result<(f32,f32,f32),String>, but we don't really know which f32 is for what. Is it (hdpi,ddpi,vdpi),(vdpi,ddpi,hdpi), or something else ? Well, of course you can see for yourself if you look at the source but I think it would be nicer if we could have that in the docs as well :)